internal/config/profile.go

```diff
@@ -196,6 +200,15 @@ func getConfigHostnameFromEnvs() string {
 	return configHostName
 }
 
+// newCLIUserTypeFromEnvs patches the user type information based on set env vars.
+func newCLIUserTypeFromEnvs() string {
+	if os.Getenv(CLIUserTypeEnv) == UniversityUser {
+		return UniversityUser
+	}
+
+	return DefaultUser
```

Comment:
to avoid adding another check when other user types are added, wdyt about:
```suggestion        
	if value, ok := os.LookupEnv(CLIUserTypeEnv); ok {
		return value
	}

	return DefaultUser
```

Reply:
oh nice 💯

Reply:
accepted! 👍  xD